spec/models/news_spec.rb

```diff
@@ -0,0 +1,60 @@
+# == Schema Information
+#
+# Table name: news
+#
+#  id         :integer          not null, primary key
+#  author     :string
+#  title      :string
+#  desc       :text
+#  source     :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+require 'rails_helper'
+
+RSpec.describe News, type: :model do
+
+  context "attributes" do
```

Comment:
INFO: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)